### DISCUSSION TITLE
JDBC tracing while using manual instrumentation

### DISCUSSION THREAD

**Discussion by kimkaesar** [open-telemetry/opentelemetry-java-instrumentation#3562] [Q&A]
Hi,

I'm currently working on a Java project that requires manual instrumentation, along with the jdbc tracing capability.
From the "About" section, I noticed that they provided JDBC standalone instrumentation library and here is the [ === Link ===](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jdbc/library)

Yet, I couldn't find the package from maven central. I wonder if anyone has any idea on this.
Thanks in advance.

**Comment by iNikem** [comment]
It was not yet released as part of the official releases. It is available as a snapshot: https://oss.sonatype.org/content/repositories/snapshots/io/opentelemetry/instrumentation/opentelemetry-jdbc/
